docs/source/setup/single-machine.rst

```diff
@@ -3,24 +3,24 @@ Single-Machine Scheduler
 
 The default Dask scheduler provides parallelism on a single machine by using
 either threads or processes.  It is the default choice used by Dask because it
-requires no setup.  You don't need to make any choices or set anything up to
-use this scheduler, however you do have a choice between threads and processes:
+requires no setup. You don't need to make any choices or set anything up to
+use this scheduler. However, you do have a choice between threads and processes:
 
 1.  **Threads**: Use multiple threads in the same process.  This option is good
     for numeric code that releases the GIL_ (like NumPy, Pandas, Scikit-Learn,
     Numba, ...) because data is free to share.  This is the default scheduler for
-    ``dask.array``, ``dask.dataframe``, and ``dask.delayed``
+    ``dask.array``, ``dask.dataframe``, and ``dask.delayed``.
 
 2.  **Processes**: Send data to separate processes for processing.  This option
     is good when operating on pure Python objects like strings or JSON-like
-    dictionary data that holds onto the GIL_ but not very good when operating
-    on numeric data like Pandas dataframes or NumPy arrays.  Using processes
-    avoids GIL issues but can also result in a lot of inter-process
+    dictionary data that holds onto the GIL_, but not very good when operating
+    on numeric data like Pandas DataFrames or NumPy arrays.  Using processes
```

Comment:
Thoughts on capitalization of DataFrames vs dataframes ?  The type name is DataFrame, but I wonder if we should use the uncapitalized version in normal prose.  This would be analagous to using "arrays"

Reply:
I don't think using the lower-case format would be the best way to refer to tabular data like Pandas' DataFrames. For instance, what is the correct way to write it? Is it "dataframes" or "data frames"? Surely "data frames" is the correct way to write it, but it does not convey the same meaning as a Pandas or Dasks DataFrame object. (Also, "dataframes" looks like a typo)

Indeed, it would be nice to use dataframes as we use arrays in normal prose, but it will come with a cost of most likely being used randomly along side DataFrames. Now, if we just bite the bullet and adopt DataFrame in camel case, it would avoid inconsistencies or random usage throughout the text and it would always be correct and consistent. Also, "DataFrames" is how it is written exclusively in the pandas docs (no exceptions for what I can see). 

If the goal is to avoid inconsistencies of this kind, I think that DataFrames would be the fool proof way.

Reply:
OK, happy to defer to the convention in the Pandas docs